src/bp/core/services/stats-service.ts

```diff
@@ -207,4 +263,67 @@ export class StatsService {
   private async getCollaboratorsCount(): Promise<number> {
     return (await this.authService.getAllUsers()).length
   }
+
+  private modulesWhitelist = {
+    builtin: true,
+    analytics: true,
+    'basic-skills': true,
+    'channel-web': true
+  }
+
+  private async getBuiltinActionsStats() {
```

Comment:
I feel like this file will become much much bigger with the growing amount of stats we want to track.

we might want to create some data collectors and register them in this stats service. the stats service job would only be to collect the data from those collector and send them to telemetry server not having to know what needs to be tracked